### DISCUSSION TITLE
GoPro Hero9 USB Power Recording losing setting

### DISCUSSION THREAD

**Discussion by mitts-hub** [gopro/labs#1169] [General]
Hello, I am trying to figure out the USB Power Trigger for my GoPro Hero9.

I can get USB Power recording to work but the issue is if I manually turn the camera on it loses this setting. I have to rescan the QR code in order to get USB Power Recording to work again. Is there something I am doing wrong or is this feature not available?

This is the code I am using:
https://gopro.github.io/labs/control/set/?cmd=!uN!S!u20E!R&title=USB%20Power%20Trigger

**Comment by dnewman-gpsw** [comment]
HERO9 might be too old of this. There is a command !MWAKE=2 this is listed as not compatible with HERO9 -- this is the command that make a camera boot when boot is applied to USB. You could try it just in case the documentation is wrong. From there you can have default start-up actions (using a BOOT command.) Everything in Labs got a lot better with HERO10 and newer.
